### PR TITLE
SES 222 feature cookie issue storybook

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
https://trello.com/c/vl1Dc2j6/222-feature-cookie-issue

# Actions
Add storybook checkbox
Add is-Light as props test storybook
Add storybook banner policy and correct props is-Light in Custom Button

### CODE REVIEW

File: src/stories/components/cookies-policy-banner/cookies-policy-banner.tsx [hunk 1]
```diff
@@ -3,11 +3,11 @@ import React from 'react';
 import lightTheme from '../../../../styles/theme/light';
 
 import { ButtonType } from '../../../core/enums/button-type.enum';
-import CheckBox from '../../components/check-box/check-box';
 import { CustomButton } from '../../components/custom-button/custom-button';
+import CheckBox from '../check-box/check-box';
 
 interface Props {
-  isShowBanner: boolean;
+  // isShowBanner: boolean;
```
Comment (YasielCabrera) [inline src/stories/components/cookies-policy-banner/cookies-policy-banner.tsx:10]: if this property is not used then should be completely removed

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/components/custom-button/custom-button.tsx [hunk 2]
```diff
@@ -95,69 +95,67 @@ export const CustomButton = ({
...
+          ? active
+            ? customStyles[buttonType].activeBorderColor
+            : customStyles[buttonType]?.borderColor
+          : customStyles[buttonType].borderColorMobile
+        : allowsHover
+        ? active
+          ? customStyles[buttonType].activeBorderColorDark
+          : customStyles[buttonType]?.borderColorDark
+        : customStyles[buttonType].borderColorMobileDark,
+      ...props.style,
```
Comment (YasielCabrera) [inline src/stories/components/custom-button/custom-button.tsx:130]: note that this added unwanted changes to the auditor comment form component. You can see the visual change on chromatic

**Review by YasielCabrera — CHANGES_REQUESTED** [review]
There are unwanted changes on other components

File: src/stories/components/check-box/check-box.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    isChecked: true,
+    label: 'Show empty budget categories',
+  },
+  {
+    isChecked: false,
+    label: 'Show empty budget categories',
+  },
+];
+
+export const [[Checked, Unchecked]] = createThemeModeVariants(CheckBox, variantsArgs);
```
Comment (YasielCabrera) [inline src/stories/components/check-box/check-box.stories.tsx:22]: Please, note that `createThemeModeVariants` return 2 stories for each item in `variantsArgs`. The expected return value for your case would be:

```js
export const [[Checked, CheckedDarkMode], [Unchecked, UncheckedDarkMode]] = createThemeModeVariants(CheckBox, variantsArgs);
```

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

**Review by YasielCabrera — APPROVED** [review]
LGTM

File: src/stories/components/cookies-policy-banner/cookies-policy-banner.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/General/CookiesPolicyBanner',
+  component: CookiesPolicyBanner,
+  parameters: {
+    layout: 'fullscreen',
+    chromatic: {
+      viewports: [1194 | 375],
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/cookies-policy-banner/cookies-policy-banner.stories.tsx:11]: not problem, fixed

**Review by YasielCabrera — APPROVED** [review]

**Pull request merged**